.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "semi": true,
+  "singleQuote": false,
```

Comment:
This should be true to reduce the number of prettier changes and make it easier to review.